src/slidepane/styles/slidePane.m.css

```diff
@@ -72,19 +100,22 @@
 	transform: translateX(0%) !important;
 }
 
-.left.slideOut {
-	transition: transform ease-in-out var(--short-animation-duration);
-}
-
-.right.slideOut {
-	transition: transform ease-in-out var(--short-animation-duration);
+.topFixed.slideInFixed {
+	transform: translateY(0%) !important;
 }
 
-.left.slideIn {
-	transition: transform ease-in-out var(--short-animation-duration);
+.bottomFixed.slideInFixed {
+	transform: translateY(0%) !important;
 }
 
-.right.slideIn {
+.left.slideOut,
```

Comment:
do you really need `.right` / `.top` etc prefixes here? Looks like you've added all of them